Roles/Setup/tasks/main.yml

```diff
@@ -0,0 +1,25 @@
+---
+# tasks file for Setup
+
+  - name: Install pip from apt
+    apt: name=python-pip state=present
+
+  - name: Update APT package cache
+    apt: update_cache=yes
+
+  - name: Run apt-get upgrade
+    apt: upgrade=full
+
+  - name: Username and Password
+      user: user
+      password: "{{password}}"
```

Comment:
Opws to eixes prin, tha exeis ousiastika enan xristi "user" se kathe vm me diaforetiko password? Me ton idio tropo mporeis na valeis kai diaforetiko xristi per host. Thes diaforetikous xristes se kathe vm?